.github/workflows/ci.yaml

```diff
@@ -231,7 +231,7 @@ jobs:
             KUBERNETES_ENV="staging"
             REPLICA_COUNT="0"
             IMAGE_GLOB="${IMAGE_TAG/$GITHUB_SHA/*}"
-            MEMORY_REQUEST="4Gi"
+            MEMORY_REQUEST="16Gi"
```

Comment:
do you still need this change? I assume it was added just for testing?

Reply:
Yes just for testing